pkg/cmd/roachtest/tests/cluster_to_cluster.go

```diff
@@ -981,11 +1003,12 @@ func runAcceptanceClusterReplication(ctx context.Context, t test.Test, c cluster
 		srcNodes: 1,
 		dstNodes: 1,
 		// The timeout field ensures the c2c roachtest driver behaves properly.
-		timeout:                   10 * time.Minute,
-		workload:                  replicateKV{readPercent: 0, debugRunDuration: 1 * time.Minute, maxBlockBytes: 1},
-		additionalDuration:        0 * time.Minute,
-		cutover:                   30 * time.Second,
-		skipNodeDistributionCheck: true,
+		timeout:                       10 * time.Minute,
+		workload:                      replicateKV{readPercent: 0, debugRunDuration: 1 * time.Minute, maxBlockBytes: 1, initWithSplitAndScatter: true},
+		additionalDuration:            0 * time.Minute,
+		cutover:                       30 * time.Second,
+		skipNodeDistributionCheck:     true,
+		nonRevisionHistoryFingerprint: true,
```

Comment:
is the fingerprint in this case also running into the PTS issue?

Reply:
in the one test I ran, it did not. The non-revision history AOST query it created took 2.5 hours, so I dont think its immune from a pts error-- a user could fingerprint a large table with a ttl policy lower than 2.5 hours for example.

Reply:
hmm but the test timeout is 10 mins, so if a fingerprint takes 2.5 hours wouldn't it just timeout?

Reply:
ah whoops. I added that as an ad hoc test. removed this flag from the acceptance test.